docs/core/extensions/generic-host.md

```diff
@@ -42,7 +44,34 @@ IHost host = builder.Build();
 host.Run();
 ```
 
-## Default builder settings
+# [IHostBuilder](#tab/hostbuilder)
+
+- Calls a <xref:Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder> method to create and configure a builder object.
+- Calls <xref:Microsoft.Extensions.Hosting.HostApplicationBuilder.Build> to create an <xref:Microsoft.Extensions.Hosting.IHost> instance.
```

Comment:
<xref:Microsoft.Extensions.Hosting.HostApplicationBuilder.Build> should be <xref:Microsoft.Extensions.Hosting.IHostBuilder.Build>